app/v1/controllers/AmendBFLossController.scala

```diff
@@ -56,6 +56,9 @@ class AmendBFLossController @Inject()(val authService: EnrolmentsAuthService,
       logger.info(message = s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] " +
         s"with correlationId : $correlationId")
 
+      logger.info(message = s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] - " +
```

Comment:
the request body in all requests needs to be removed it is a security risk

Reply:
Yh it is something I highlighted but was ignored so just went with what was in the description